### PR TITLE
Add YakovGTest Class

### PR DESCRIPTION
**Pull request by iadigar**
First commit

### CODE REVIEW

**Review by SergeiKuznetsov — APPROVED** [review]

File: src/test/java/YakovGTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void beforeTest() {
+        driver = new ChromeDriver();
+    }
+
+    @Test
+    public void test() throws InterruptedException {
```
Comment (GalinaRuban) [inline src/test/java/YakovGTest.java:23]: If there is no sleep in the test, this part throws InterruptedException is not needed.

File: src/test/java/YakovGTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        driver.get("https://www.sportsmans.com/");
+
+        WebElement input = driver.findElement(By.id("js-site-search-input"));
+        input.sendKeys("FN Five SeveN Pistol\n");
+
+        WebElement result = driver.findElement(By.xpath("/html/body/main/div[4]/div[1]/div/div[2]/div/div/div[1]/ul/li[1]/div/div[2]/div[2]/h2/a"));
```
Comment (GalinaRuban) [inline src/test/java/YakovGTest.java:29]: This xpath is too long. Can be replaced with //h2/a[@data-product-id="p41796"]

File: src/test/java/YakovGTest.java (hunk 1 above)
Comment (iadigar) [reply src/test/java/YakovGTest.java]: Done

File: src/test/java/YakovGTest.java (hunk 2 above)
Comment (iadigar) [reply src/test/java/YakovGTest.java]: Done

**Review by GalinaRuban — APPROVED** [review]

**Pull request merged**